quickquickstarts/Using_Validators_with_a_Windows_image/validators_with_a_windows_image.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Please keep the filename consistent with existing naming scheme.
If current scheme doesn't seem okay to you then please change the scheme for all of them - but the customer should have a consistent experience.